adaptive_components/lib/src/adaptive_column.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+import 'package:flutter/material.dart';
+import 'adaptive_container.dart';
+
+/// The [AdaptiveColumn] follows the Material guideline for responsive grids.
+///
+/// https://material.io/design/layout/responsive-layout-grid.html#columns-gutters-and-margins
+///
+/// This widget intigrates with the [AdaptiveContainer] widget.
+/// The [AdaptiveContainer] widget has a parameter called columns which represents
+/// the amount of columns it should take according to the breakpoints package.
+///
+/// So if the user has 6 adaptive container and each container represents two columns
+/// then the 6 adaptive container would all fit within one Row on a extra large screen
+/// because extra large screens have 12 columns per row.
+///
+/// Learn more about the breakpoint system:
+///
+/// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+class AdaptiveColumn extends StatelessWidget {
+  /// Creates a vertical array of children. Going from left to right and then
+  /// top to bottom.
+  ///
+  /// To see an example visit:
+  /// https://adaptive-components.web.app/#/
+  const AdaptiveColumn({
+    this.gutter,
+    this.margin,
+    @required this.children,
+  }) : assert(children != null);
+
+  /// Empty space at the left and right of this widget.
+  final double margin;
+
+  /// Represents the space between children.
+  final double gutter;
+  final List<AdaptiveContainer> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (BuildContext context, BoxConstraints constraints) {
+        BreakpointSystemEntry _entry = getBreakpointEntry(context);
+        final double _margin = margin ?? _entry.margin;
+        final double _gutter = gutter ?? _entry.gutter;
+        final int _totalColumns = _entry.columns;
+
+        return Container(
+          width: MediaQuery.of(context).size.width - (_margin * 2),
```

Comment:
nit: parenthesis not needed

Reply:
If the margin is being accounted for in the `margin` property, why would the width need to be further adjusted with the _margin value?

Reply:
Remove width